Pages/Result/result.css

```diff
@@ -0,0 +1,19 @@
+.result-grid {
+  display: grid;
+  padding: var(--padding);
+  flex-direction: column;
```

Comment:
How can we use `display:grid` and `flex-direction:column` on same element. Isn't flex-direction only specific to flex display? I am little confused here.

Reply:
Oh sorry, previously i used flex so I forgot to remove that. you are right.